src/emulator/dataconnectEmulator.ts

```diff
@@ -61,21 +64,51 @@ export class DataConnectEmulator implements EmulatorInstance {
     } catch (err: any) {
       this.logger.log("DEBUG", `'fdc build' failed with error: ${err.message}`);
     }
-    return start(Emulators.DATACONNECT, {
-      auto_download: this.args.auto_download,
-      listen: listenSpecsToString(this.args.listen),
-      config_dir: this.args.configDir,
-      project_id: this.args.projectId,
-      service_location: this.args.locationId,
-    });
+    const alreadyRunning = await this.discoverRunningInstance();
+    if (alreadyRunning) {
+      this.logger.log(
+        "INFO",
```

Comment:
nit: could we use something like   `hubLogger.logLabeled("BULLET", "emulators",  "Deteced an instance...");`


this message is surrounded by other louder warnings, hard to see.